src/frontend/src/binder/insert.rs

```diff
@@ -37,6 +37,8 @@ pub struct BoundInsert {
     /// Used as part of an extra `Project` when the column types of `source` query does not match
     /// `table_source`. This does not include a simple `VALUE`. See comments in code for details.
     pub cast_exprs: Vec<ExprImpl>,
+
+    pub returning_list: (Vec<ExprImpl>, Vec<Option<String>>),
```

Comment:
Seems the second item of aliases is not ever used. May refer to here for how it affects.

https://github.com/risingwavelabs/risingwave/blob/cc4b10ced6080552b3f14ebc3b435c6cd5c16c8f/src/frontend/src/binder/select.rs#L152-L160

The names of the output columns should be:
- if there's alias, use the alias
- or use the column name of the table, instead of the child node like `VALUES`